theme.css

```diff
@@ -2172,4 +2172,36 @@ li[data-task="b"]>input:checked,
 li[data-task="b"]>p>input:checked {
 	color: rgba(var(--color-orange-rgb), 1);
 	-webkit-mask-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' class='h-5 w-5' viewBox='0 0 20 20' fill='currentColor'%3E%3Cpath d='M5 4a2 2 0 012-2h6a2 2 0 012 2v14l-5-2.5L5 18V4z' /%3E%3C/svg%3E");
-}
\ No newline at end of file
+}
+
+
+/* Add Obsidian Icon on new Tab */
+/* Copied from Cupertino Theme :P */
+.empty-state-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0;
+    margin-top: var(--view-header-height);
+    padding: var(--file-margins);
+    width: 100%;
+    max-width: 100%;
+    height: 100%;
+    max-height: 100%;
+}
+
+.theme-mod-composer .workspace-leaf-content[data-type=empty] .empty-state .empty-state-container::before {
```

Comment:
`.theme-mod-composer` should follow composer's name conventions, like `composer--EnableCustomizedEmptyStateIcon`.

or if you want this feature to be enabled by default,  remove  `.theme-mod-composer`